themes/OneHalfLight.conf

```diff
@@ -1,8 +1,9 @@
-# Onehalf Light Colorscheme for Kitty
-# Based on https://github.com/sonph/onehalf
-# By https://github.com/dbinary
-
-macos_titlebar_color #FAFAFA
```

Comment:
Note: I went ahead and removed `macos_titlebar_color` from One Half Light because I believe it is too opinionated to belong upstream.